cross/jdupes/Makefile

```diff
@@ -13,10 +13,8 @@ COMMENT  = A powerful duplicate file finder and an enhanced fork of fdupes.
 LICENSE  = MIT
 
 CONFIGURE_TARGET = nop
-INSTALL_TARGET = jdupes_install
+INSTALL_MAKE_OPTIONS = install DESTDIR=$(INSTALL_DIR) PREFIX=$(INSTALL_PREFIX)
 
-include ../../mk/spksrc.cross-cc.mk
+ADDITIONAL_CFLAGS = -DENABLE_DEDUPE
```

Comment:
doesnt work (tested it)

Reply:
Set instead ```COMPILE_MAKE_OPTIONS = ENABLE_DEDUPE=1``` and it does. Tested via:
```
root@(...):/spksrc/spk/synocli-file# ./work-x64-7.1/jdupes-1.21.0/jdupes --version
jdupes 1.21.0 (2022-09-03) 64-bit
Compile-time extensions: fsdedup
```

Reply:
for me it works as implemented in this PR.

aarch64-6.1 package downloaded from github build action and installed on DS218 with DSM 6.2.4 shows:
```
$ jdupes --version
jdupes 1.21.0 (2022-09-03) 64-bit
Compile-time extensions: fsdedup
Copyright (C) 2015-2022 by Jody Bruchon and contributors
Forked from fdupes 1.51, (C) 1999-2014 Adrian Lopez and contributors
....
```

you can download the packages created by github build action under "checks - build - artifacts".

Reply:
ah, I tried only the COMPILE_MAKE_OPTIONS without the rest of changes I suppose is why. My bad!

Reply:
@jisakiel can you please test the dedup feature ❔ 

I am not sure, whether this version is fully self contained to dedup files. 
I have read that btrfs libraries are needed and I like to have a confirmation that no external library is missing in jdupes for dedup to work.

You can use the spk file created by github build action (see above how to download).

Reply:
It works only on appearance :(

```
jisakiel@diskstation:~$ jdupes --version | head
jdupes 1.21.0 (2022-09-03) 64-bit
Compile-time extensions: fsdedup
```

But:
```
jdupes --dedupe -1 -r /volume1/Backups/*
...
  [SRC] /volume1/Backups/(...)/(...).jpg
  -XX-> /volume1/Backups/(...)/(...)/(...)
error: Inappropriate ioctl for device (25)
```

Stracing around a bit with some sketchy static strace binary I found gets me not a whole lot of clarity more:
```
ioctl(3, _IOC(_IOC_READ|_IOC_WRITE, 0x5e, 0x36, 0x18), 0x1b0e4f0) = -1 ENOTTY (Inappropriate ioctl for device)
```

Reply:
Tested this on https://github.com/SynoCommunity/spksrc/commit/c932c4a326edad6efcf8047eeb704c27166b4006 (I don't think I can update this pr with it) and it works now:

```
jdupes --dedupe -1 -r /volume1/Backups/*
...
  [SRC] /volume1/Backups/(...).dng
  ====> /volume1/Backups/(...)/Copy(...).dng

Deduplication done (18 files processed)
```

Reply:
@jisakiel thanks for testing this. I have applied the patch.

Just looking for updates of other tools in synocli-file, before building a new release...